__tests__/commands/install/bin-links.js

```diff
@@ -20,6 +21,36 @@ async function linkAt(config, ...relativePath): Promise<string> {
   }
 }
 
+function execCommand(cwd: string, binPath: Array<string>, args: Array<string>): Promise<Array<?string>> {
```

Comment:
Yeah I copied it over from the test file that was testing `run` command and tweaked it for my purpose. I think it was out of scope for this diff but the right thing to do is probably have a place to store and dedupe all of these similar functions (you're right--I've definitely seen it being reused in several other test files!) inside some shared test utils folder.

On a separate note, that looks like a cool tool! I think I've seen similar functions under src/ which is probably more important to refactor :)